spec/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'order'
+
+describe Order do 
+
+  it 'prints the menu' do
+    expect(subject.print_menu).to include("Sushi set" => 12)  
+  end
+
+  it 'tells user that an item is not on the menu' do
+    expect { subject.user_order("cake") }.to output("Item not on menu\n").to_stdout
```

Comment:
Nice test for this branch of the `user_order` method! To achieve even better test coverage, we'll need to test all possible branches of this method 🙂